bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Bookie.java

```diff
@@ -149,6 +150,8 @@
     private volatile boolean running = false;
     // Flag identify whether it is in shutting down progress
     private volatile boolean shuttingdown = false;
+    // Bookie status
+    private BookieStatus bookieStatus = new BookieStatus();
```

Comment:
`bookieStatus` can be made `final` as it its object reference is not expected to change for the life of the Bookie (but the state will).